tests/std/tests/P0896R4_views_join/test.cpp

```diff
@@ -504,8 +518,45 @@ int main() {
         assert(ranges::equal(joined, result));
     }
 
-#if defined(__clang__) || defined(__EDG__) // TRANSITION, VSO-934264
+    { // P2328 range of prvalue array
+        static constexpr int result[] = {1, 2, 3, 4, 5};
+        auto ToArray                  = [](const int i) { return array<int, 1>{i + 1}; };
+        assert(ranges::equal(views::iota(0, 5) | views::transform(ToArray) | views::join, result));
+        static_assert(ranges::equal(views::iota(0, 5) | views::transform(ToArray) | views::join, result));
+    }
+
+    { // P2328 range of prvalue vector using global function
+        static constexpr int result[] = {1, 2, 3, 4, 5};
+        assert(ranges::equal(views::iota(0, 5) | views::transform(ToVector) | views::join, result));
+#if defined(__clang__) || defined(__EDG__) || defined(MSVC_INTERNAL_TESTING) // TRANSITION, VSO-934264

```

Comment:
I'm still confused about these uses of `MSVC_INTERNAL_TESTING`, given that VSO-934264 doesn't seem to have been fixed.

Reply:
For other reviewers: I validated this passes with the internal compiler (64-bit, at least). Probably the bug has been fixed by a related change and the database just hasn't caught up yet.

Reply:
I observe that this is failing in the internal test harness (x86chk and amd64chk). I'm unsure what the difference was (did you define `MSVC_INTERNAL_TESTING`?) but I believe we'll need to drop this coverage for MSVC.